packages/utils/src/lib/git.ts

```diff
@@ -17,3 +17,40 @@ export async function getLatestCommit() {
   });
   return log.latest;
 }
+
+export async function branchHasChanges(): Promise<boolean> {
+  return await git.status(['-s']).then(r => r.files.length > 0);
+}
+
+export async function guardAgainstDirtyRepo(): Promise<void> {
```

Comment:
Similarly here, maybe it's just me but I would not know what a dirty repo is.

```suggestion
export async function guardAgainstLocalChanges(): Promise<void> {
```